src/components/Editor/index.tsx

```diff
@@ -138,16 +135,21 @@ const Editor = () => {
     const {mode, systemMode} = useColorScheme();
 
     const beginLineNumToLogEventNum = useViewStore((state) => state.beginLineNumToLogEventNum);
+    const {isPrettified} = useViewStore.getState();
+    const updateIsPrettified = useViewStore((state) => state.updateIsPrettified);
+
     const logData = useViewStore((state) => state.logData);
+
+    const logEventNum = useViewStore((state) => state.logEventNum);
+    const {setLogEventNum} = useViewStore.getState();
```

Comment:
```suggestion
```
see the suggestion about retrieving the action directly inside the useCallback wrapped function